ts/Core/Chart/Chart.ts

```diff
@@ -2324,26 +2324,37 @@ class Chart {
 
         // Apply new links
         chartSeries.forEach(function (series): void {
-            let linkedTo = series.options.linkedTo;
+            const { linkedTo } = series.options;
 
-            if (isString(linkedTo)) {
+            if (linkedTo && isString(linkedTo)) {
+                let seriesLinkedTo: Series | undefined;
```

Comment:
Instead of calling the local variable `seriesLinkedTo`, I think it should be called `linkedParent` in consistency with the series property.